sql/check_duplicated_foreign_keys.sql

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2019-2024. Ivan Vakhrushev and others.
+ * https://github.com/mfvanek/pg-index-health-sql
+ *
+ * Licensed under the Apache License 2.0
+ */
+
+with
+-- combine fk constraints with the attributes involved in them
+fk_with_attributes as (
+        select
+            c.conname as fk_name,
+            c.conrelid,
+            c.confrelid,
+            fk_conkey.conkey_order as att_order,
+            fk_conkey.conkey_number,
+            fk_confkey.confkey_number,
+            rel_att.attname as rel_att_name,
+            rel_att.atttypid as rel_att_type_id,
+            rel_att.atttypmod as rel_att_type_mod,
+            rel_att.attnotnull as rel_att_notnull,
+            frel_att.attname as frel_att_name,
+            frel_att.atttypid as frel_att_type_id,
+            frel_att.atttypmod as rel_att_type_mod,
+            frel_att.attnotnull as rel_att_notnull
+        from pg_catalog.pg_constraint as c
+            cross join lateral unnest(c.conkey) with ordinality as fk_conkey(conkey_number, conkey_order)
+            left join lateral unnest(c.confkey) with ordinality as fk_confkey(confkey_number, confkey_order)
+                on fk_conkey.conkey_order = fk_confkey.confkey_order
+            left join pg_catalog.pg_attribute as rel_att
+                on rel_att.attrelid = c.conrelid and rel_att.attnum = fk_conkey.conkey_number
+            left join pg_catalog.pg_attribute as frel_att
+                on frel_att.attrelid = c.confrelid and frel_att.attnum = fk_confkey.confkey_number
+        where c.contype in ('f')
+    ),
+    --
+    fk_with_attributes_grouped as (
+        select
+            fk_name,
+            conrelid,
+            confrelid,
+            array_agg (rel_att_name order by att_order) as rel_att_names,
+            array_agg (frel_att_name order by att_order) as frel_att_names
+        from fk_with_attributes
+        group by 1, 2, 3
+    )
+select
+    r_from.relname,  -- referencing relation
```

Comment:
We have a domain model in java code for Foreign Keys https://github.com/mfvanek/pg-index-health/blob/master/pg-index-health-model/src/main/java/io/github/mfvanek/pg/model/constraint/ForeignKey.java

It is necessary to use the same naming as in https://github.com/mfvanek/pg-index-health-sql/blob/master/sql/foreign_keys_without_index.sql#L12